nuxt/.env.template

```diff
@@ -1 +1,3 @@
-NUXT_PUBLIC_GOOGLE_ANALYTICS_ID=
+NUXT_TURNSTILE_SECRET_KEY=
```

Comment:
This should passed as a [Docker secret](https://docs.docker.com/engine/swarm/secrets/).

Reply:
WIP: https://github.com/nuxt-modules/turnstile/pull/174

Reply:
I've update nuxt-turnstile to use v0.2.0 which incorporates your PR

Reply:
`NUXT_TURNSTILE_SECRET_KEY` can be removed, right?